.env.example

```
@@ -120,5 +120,18 @@ AUTOMATIC_LANGFUSE_SYSTEM_PROMPT_LABEL=automatic_system_langfuse_prompt
+# Context Summarization Configuration
+# Controls conversation context management and memory optimization
+ENABLE_SUMMARIZATION=true
+MAX_TURNS_BEFORE_SUMMARY=1
+KEEP_RECENT_TURNS=1
+
+# Local Tools Fallback Configuration
+# Critical for MCP integration reliability - enables fallback to local tools when MCP server fails
+# Set to true for production environments to ensure system availability
+ENABLE_LOCAL_TOOLS_FALLBACK=false
+# When true, disables local system tools (get_current_time, etc.) to use only NeuroLink MCP tools
+DISABLE_LOCAL_SYSTEM_TOOLS=true
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DISABLE_LOCAL_SYSTEM_TOOLS key should go before the ENABLE_LOCAL_TOOLS_FALLBACK key

(UnorderedKey)